ui/src/org/pentaho/di/ui/trans/steps/ivwloader/IngresVectorwiseLoaderDialog.java

```diff
@@ -404,9 +403,7 @@ public void widgetDefaultSelected( SelectionEvent event ) {
   }
 
   protected TextVar addPasswordTextVar( String labelMessageKey, Control prevControl ) {
-    TextVar textVar = addStandardTextVar( labelMessageKey, prevControl );
```

Comment:
As above - this needs to be reverted - this is the only place  PasswordTextVar should be created.